toothpick-runtime/src/main/java/toothpick/ScopeNode.java

```diff
@@ -276,4 +280,9 @@ private void checkIsAnnotationScope(Class<? extends Annotation> scopeAnnotationC
   private boolean isScopeAnnotationClass(Class<? extends Annotation> scopeAnnotationClass) {
     return scopeAnnotationClass.isAnnotationPresent(javax.inject.Scope.class);
   }
+
+  @Override
+  public void setClosedListener(ScopeClosedListener closeListener) {
+    this.closeListener = closeListener;
```

Comment:
does it not lead to bad programming practices? -> Using this to know when activity is destroyed for instance.